File: java/src/main/java/com/example/doubleup/dto/PlayStartRequestDto.java
package com.example.doubleup.dto;

public class PlayStartRequestDto extends PlayContinueRequestDto {
    private Long betSize;

    public PlayStartRequestDto() {}

    public Long getBetSize() {
        return betSize;
    }

    public void setBetSize(Long betSize) {
        this.betSize = betSize;
    }
}

File: go/dto/play_start_request_dto.go
package dto

import (
    "github.com/google/uuid"
    "double_up/enums"
)

type PlayStartRequestDto struct {
    PlayerID uuid.UUID `json:"PlayerID"`
    Choice   enums.SmallLargeChoice `json:"Choice"`
    BetSize int64 `json:"BetSize"`
}
